src/components/CivicLikerPage/sections/SuggestedCreatorsSection.vue

```diff
@@ -10,6 +10,7 @@
         preset="creator"
         :contents="contents"
         :is-waterfall-layout="false"
+        :is-loading="contents.length === 0"
```

Comment:
Does there exists a case `contents` is really of length 0?

Reply:
I think not, contents should be covered by suggested API if personal API return empty